### PR TITLE
[AMS] Update AMS module to 8.0 Python sdk

### PR DESCRIPTION
**Pull request by hivyas**
**Description**

Updating AMS module to the latest Python sdk (8.0) 
Changes include:
• `az ams transform create` has a new parameter blur_type
• `az ams account create` has new parameters for managed identities
• `az ams account create` has new parameters for key delivery
• `az ams account create` has a new parameter public_network_access
• `az ams account encryption` has new parameters for managed identities
•  `az ams account encryption` has new parameters managed identities

**Testing Guide**

`az ams account identity assign -n myAmsAccount -g myRG --user-assigned myUserId`
`az ams account create -n myAmsAccount -g myRG --storage-account myStorageAccount --location location --mi-system-assigned  --default-action Allow --public-network-access`

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AMS

File: src/azure-cli/azure/cli/command_modules/ams/_params.py [hunk 1]
```diff
@@ -66,7 +70,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('ams account create') as c:
         c.argument('storage_account', storage_account_arg_type,
                    help='The name or resource ID of the primary storage account to attach to the Azure Media Services account. The storage account MUST be in the same Azure subscription as the Media Services account. It is strongly recommended that the storage account be in the same resource group as the Media Services account. Blob only accounts are not allowed as primary.')
-        c.argument('assign_identity', options_list=['--mi-system-assigned'], action='store_true', help='Set the system managed identity on the media services account.')
+        c.argument('mi_system_assigned', action='store_true', help='Set the system managed identity on the media services account.', arg_group='Identity')
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/ams/_params.py:73]: Why not use `arg_type=get_three_state_flag()` for this argument?

File: src/azure-cli/azure/cli/command_modules/ams/operations/encryption.py [hunk 2]
```diff
@@ -17,19 +18,35 @@ def get_encryption(client, resource_group_name, account_name):
...
+            encryption = AccountEncryption(type=key_type, key_vault_properties=None, identity=None)
 
+        media_service: MediaService = MediaService(location=account_info.location,
+                                                   storage_accounts=account_info.storage_accounts,
+                                                   key_delivery=account_info.key_delivery,
+                                                   identity=account_info.identity, encryption=encryption,
+                                                   storage_authentication=account_info.storage_authentication,
+                                                   name=account_info.name,
+                                                   public_network_access=account_info.public_network_access)
+        print(encryption)
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/ams/operations/encryption.py:49]: Do you need to remove this line?
```suggestion
        print(encryption)
```

File: src/azure-cli/azure/cli/command_modules/ams/operations/job.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ def create_job(client, resource_group_name, account_name, transform_name, job_na
                label=None, correlation_data=None,
                description=None, priority=None, files=None, base_uri=None):
     from azure.mgmt.media.models import (Job, JobInputAsset, JobInputHttp)
-
+    print(files)
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/ams/operations/job.py:16]: Do you need to remove this print?

**Review by kairu-ms — APPROVED** [review]

**Comment by jiasli** [comment]
This PR breaks CI:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1469597&view=logs&j=74095127-2a27-5370-37ed-15a4193f243f&t=5cf5f89a-09fb-583c-72e4-4e4427c89fb1&l=6643

```
2022-03-30T04:59:29.0941877Z         except CannotOverwriteExistingCassetteException as ex:
2022-03-30T04:59:29.0942329Z >           raise AssertionError(ex)
2022-03-30T04:59:29.0943442Z E           AssertionError: Can't overwrite existing cassette ('/home/vsts/work/1/s/src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_media_service.yaml') in your current record mode ('once').
2022-03-30T04:59:29.0945091Z E           No match for the request (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test_private_endpoint_connection_media_service000001/providers/Microsoft.Media/mediaservices/clitestams000003?api-version=2021-06-01>) was found.
2022-03-30T04:59:29.0946233Z E           Found 1 similar requests with 1 different matcher(s) :
2022-03-30T04:59:29.0946774Z E           
2022-03-30T04:59:29.0947927Z E           1 - (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test_private_endpoint_connection_media_service000001/providers/Microsoft.Media/mediaservices/clitestams000003?api-version=2021-05-01>).
2022-03-30T04:59:29.0949036Z E           Matchers succeeded : ['method', 'scheme', 'host', 'port', 'path']
2022-03-30T04:59:29.0949523Z E           Matchers failed :
2022-03-30T04:59:29.0950133Z E           _custom_request_query_matcher - assertion failure :
2022-03-30T04:59:29.0950578Z E           None
2022-03-30T04:59:29.0950781Z 
2022-03-30T04:59:29.0951389Z src/azure-cli-testsdk/azure/cli/testsdk/base.py:299: AssertionError
2022-03-30T04:59:29.0952179Z - generated xml file: /home/vsts/.azdev/env_config/home/vsts/work/1/s/env/test_results.xml -
2022-03-30T04:59:29.0952804Z =========================== short test summary info ============================
2022-03-30T04:59:29.0955222Z FAILED src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py::NetworkPrivateLinkScenarioTest::test_private_endpoint_connection_media_service
```

**Pull request merged**